src/components/VmConsole/index.js

```diff
@@ -121,7 +122,7 @@ const VmConsole = ({
     if (isFullScreen) {
       onFullScreen()
     }
-  }, [isFullScreen])
+  }, [isFullScreen, fullScreenNoVnc])
```

Comment:
it seems we can remove `fullScreenNoVnc` from dependency list at it is used only to set initial value to  `isFullScreen`

Reply:
Only time I can think of when the component would need to be sensitive to a change to `fullScreenNoVnc` would be if the options change while the console is displayed.  But even then, there would need to be more changes to pick up that change.

@hstastna - I'm ok with `VmConsole` just using the initial value of `fullScreenNoVnc` and not being sensitive to any changes.  It would be weird to have the full screen option get picked up from another tab (or rest call) and have the console toggle between modes all by itself!

Reply:
Right, thanks, folks! Seems that we really don't need `fullScreenNoVnc` here, at least for now :D I was thinking what if this value changed while the console being displayed, but as Scott has mentioned, that would require some more changes ;)